course/utils.py

```diff
@@ -1320,4 +1324,215 @@ def _render_notebook_from_source(
 
 # }}}
 
+
+class RelateCsvHandler(object):
+    # Tests ref tests.template_backends.test_django.DjangoTemplatesTests
```

Comment:
I have no idea what this does and why it belongs here.

Reply:
Why is this not in `checks.py`?

Reply:
It could stand renaming to `RelateCSVSettingsValidator` or some such maybe?

Reply:
> I have no idea what this does and why it belongs here.
It's something that make the a csv configuration (currently from csv export) from settings.RelateCSVSetting, by check and add some default value if not configured, and packaged some methods like generate the user attribute choice fields (in the ExportCsvForm) for csv export.

Reply:
> Why is this not in `checks.py`?

As mentioned above, it's functionalities are not limited to check only, do you prefer to put that in `checks.py`? Won't it look weird to import a function from `checks.py`?.

Reply:
>It could stand renaming to RelateCSVSettingsValidator or some such maybe?

What about `RelateCSVSettingsInitializer`

Reply:
docstring added.